native/core/src/execution/shuffle/shuffle_writer.rs

```diff
@@ -1570,13 +1588,41 @@ pub fn write_ipc_compressed<W: Write + Seek>(
     // write ipc_length placeholder
     output.write_all(&[0u8; 8])?;
```

Comment:
I know this isn't new code on this PR, but it jumped out at me: can this be a seek_relative? For example:
```
// Seek past placeholder for ipc_length.
output.seek_relative(8)?;
```
I tried it and it seemed to work but I'm not sure I ran all of the necessary tests.